### PR TITLE
8th tutorial video review

### PR DESCRIPTION
**Pull request by Tedoshiii**
- Please use the following link for reviews: https://murf.ai/share/lmxpv0ix
- Applied the requested fixes mentioned in issue #152

### CODE REVIEW

**Comment by TallTed** [comment]
Things get renumbered. Far better to refer to the full video title than to its potentially variable position in a list.

File: Tutorials/whatiswot/8-JSON_Schema_in_Practice_3/8-JSON-Schema-3.txt [hunk 1]
```diff
@@ -1,48 +1,48 @@
 In the previous videos of JSON Schema, we have explained the features and basic implementations of JSON Schema. In this video, we will show the actual validation process with libraries and code examples.
 
-Throughout this video, we will mainly focus on validation with the Python library jsonschema and the JavaScript library AJV, but there are many other libraries in different programming languages and tools to validate JSON using a JSON Schema.
+Throughout this video, we will mainly focus on validation with the Python library jsonschema and the JavaScript library AJV, but there are many other libraries in different programming languages and tools to validate JSON using a JSON Schema.
```
Comment (TallTed) [inline Tutorials/whatiswot/8-JSON_Schema_in_Practice_3/8-JSON-Schema-3.txt:3]:
```suggestion
Throughout this video, our main focus will be on validation with the `jsonschema` Python library and the `AJV` JavaScript library, but many other libraries to validate JSON using a JSON Schema exist in different programming languages and tools. 
```

File: Tutorials/whatiswot/8-JSON_Schema_in_Practice_3/8-JSON-Schema-3.txt [hunk 2]
```diff
@@ -1,48 +1,48 @@
 In the previous videos of JSON Schema, we have explained the features and basic implementations of JSON Schema. In this video, we will show the actual validation process with libraries and code examples.
 
-Throughout this video, we will mainly focus on validation with the Python library jsonschema and the JavaScript library AJV, but there are many other libraries in different programming languages and tools to validate JSON using a JSON Schema.
+Throughout this video, we will mainly focus on validation with the Python library jsonschema and the JavaScript library AJV, but there are many other libraries in different programming languages and tools to validate JSON using a JSON Schema. 
 
-Here are some of the popular ones in different programming languages.  There are many more validators. 
+Here are some of the popular ones in different programming languages. There are many more validators.
```
Comment (TallTed) [inline Tutorials/whatiswot/8-JSON_Schema_in_Practice_3/8-JSON-Schema-3.txt:5]:
```suggestion
There are many validators. Here are some of the more popular ones.  
```

File: Tutorials/whatiswot/8-JSON_Schema_in_Practice_3/8-JSON-Schema-3.txt [hunk 3]
```diff
@@ -1,48 +1,48 @@
...
+Throughout this video, we will mainly focus on validation with the Python library jsonschema and the JavaScript library AJV, but there are many other libraries in different programming languages and tools to validate JSON using a JSON Schema. 
 
-Here are some of the popular ones in different programming languages.  There are many more validators. 
+Here are some of the popular ones in different programming languages. There are many more validators.  
 
 For further information, you can check the JSON Schema homepage via the link in the video description.
 
 This  is the basic validation flow that all libraries follow.
-The library takes a schema for your JSON data and validates your data according to the schema. Outputs the validation result as a boolean 
+The library takes a schema for your JSON data and validates your data according to the schema. Outputs the validation result as a boolean
```
Comment (TallTed) [inline Tutorials/whatiswot/8-JSON_Schema_in_Practice_3/8-JSON-Schema-3.txt:10]:
```suggestion
The library takes a schema for your JSON data and validates your data according to the schema. The validation result is output as a boolean.
```

File: Tutorials/whatiswot/8-JSON_Schema_in_Practice_3/8-JSON-Schema-3.txt [hunk 4]
```diff
@@ -1,48 +1,48 @@
...
 This  is the basic validation flow that all libraries follow.
-The library takes a schema for your JSON data and validates your data according to the schema. Outputs the validation result as a boolean 
+The library takes a schema for your JSON data and validates your data according to the schema. Outputs the validation result as a boolean  
 
 So let's start with AJV! It is a JavaScript library.
 
 This is the same temperature sensor example we used before. Let’s examine this in JavaScript code.
 
-First, We are importing the AJV. 
+First, We are importing the AJV.
```
Comment (TallTed) [inline Tutorials/whatiswot/8-JSON_Schema_in_Practice_3/8-JSON-Schema-3.txt:16]:
```suggestion
First, we import the `AJV` library.  
```

**Review by TallTed — CHANGES_REQUESTED** [review]
Small, editorial

**Comment by egekorkan** [comment]
@TallTed thank you for the reviews. We just talked about it today and @Tedoshiii will incorporate them.
Otherwise, the original issue is now resolved and I am marking it in #152

**Comment by egekorkan** [comment]
#169 takes over the changes here

**Pull request closed without merging**